maas/resource_maas_boot_source_test.go

```diff
@@ -18,14 +18,14 @@ const defaultURL = "http://images.maas.io/ephemeral-v3/stable/"
 const snapKeyring = "/snap/maas/current/usr/share/keyrings/ubuntu-cloudimage-keyring.gpg"
 
 func TestAccResourceMAASBootSource_basic(t *testing.T) {
-
 	var bootsource entity.BootSource
+
 	url := "http://images.maas.io/ephemeral-v3/candidate/"
 
 	checks := []resource.TestCheckFunc{
 		testAccMAASBootSourceCheckExists("maas_boot_source.test", &bootsource),
 		resource.TestCheckResourceAttr("maas_boot_source.test", "url", url),
-		resource.TestCheckResourceAttr("maas_boot_source.test", "keyring_filename", snapKeyring),
+		resource.TestCheckResourceAttr("maas_boot_source.test", "keyringFilename", snapKeyring),
```

Comment:
Same question if "keryingFilename" string value can be a breaking change?

Reply:
same explanation as above